mozilla-central-patches/eslintignore.diff

```diff
@@ -0,0 +1,9 @@
+diff --git a/.eslintignore b/.eslintignore
+--- a/.eslintignore
++++ b/.eslintignore
+@@ -72,3 +72,4 @@ browser/extensions/pocket/content/panels/js/vendor/**
+ browser/locales/**
+-# vendor library files in activity-stream
++# generated or library files in activity-stream
++browser/extensions/activity-stream/data/content/activity-stream.bundle.js
```

Comment:
Are we expecting other bundles? Or js files in that data/content directory? Maybe just ignore that whole directory? The explicit `bundles` subdirectory would make it clearer why it's being ignored though.